plutus-benchmark/README.md

```diff
@@ -55,32 +46,46 @@ This directory contains three sets of benchmarks:
      minutes to run the entire suite (again, this will depend on the hardware).
 
    * There are also two variants (`validation-decode` & `validation-full` ) that measure
-     the peformance of the validation dataset for the script deserialization,
-     and for the deserialization+extrachecks+execution, respectively.
+     the performance of the validation dataset for the script deserialization,
+     and for the deserialization and extra checks and execution, respectively.
+
+See also [validation/README.md](./validation/README.md).
 
 * `lists`: some simple algorithms on lists.  See [lists/README.md](./lists/README.md) for more information.
 
-See also [nofib/README.md](./nofib/README.md) and [validation/README.md](./validation/README.md).
+* `marlowe`(TODO): 
 
-### nofib-exe
+   * The source for a minimal version of the Marlowe validators is in `marlowe/src`.
+   * There is an executable in `marlowe/exe` which can be used to run benchmarks for a set of scripts.
+   * The benchmarking code is in `marlowe/bench`.
+
+   * To run the benchmarks using cabal, type a command like this
+       * `cabal bench plutus-benchmark:marlowe` or
+       * `cabal run plutus-benchmark:marlowe`
+
+See also [marlowe/README.md](./marlowe/README.md).
+
+### Executables
 The `nofib-exe` program in `nofib/exe` allows you to run the `nofib` benchmarks from the command line and
 output Plutus Core versions in a number of formats.  See the built-in help information
 for details.
 
+For `marlowe`, the `marlowe-validators` program allows you to run benchmarks in certain folders and get a summary of results. See the [marlowe/README.md](./marlowe/README.md) for detail.
+
 ### Criterion output
 
-Both sets of benchmarks will generate a file called `report.html` containing
+The benchmarks will generate a file called `report.html` containing
 detailed information about the results of running the benchmarks. This will be
 written to the `plutus-benchmark` directory.  To put it elsewhere, pass
 Criterion the `--output` option along with an *absolute* path (relative paths
 are interpreted relative to `plutus-benchmark` when running the benchmarks via
-stack or cabal): for example
+cabal): for example
 
 ```
-  stack bench plutus-benchmark:validation --ba "crowdfunding -L60 --output $PWD/crowdfunding-report.html"
+  cabal bench plutus-benchmark:validation --ba "crowdfunding -L60 --output $PWD/crowdfunding-report.html"
 ```
 
-If using `cabal bench` you'll have to do something similar, but `cabal run` will write the output into
+`cabal run` will write the output into
 the current directory (unless you use `cabal run <benchmark> -- --output ...`).
```

Comment:
@effectfully I'm not sure about the difference between `run` and `bench` but it said here `run` will write the output into the current directory. `bench` writes in the `plutus-benchmark` directory by default.